src/SFA.DAS.AANHub.Application/CalendarEvents/Queries/GetCalendarEvent/GetCalendarEventByIdQueryResult.cs

```diff
@@ -0,0 +1,23 @@
+﻿using SFA.DAS.AANHub.Application.Models;
+using SFA.DAS.AANHub.Domain.Entities;
+
+namespace SFA.DAS.AANHub.Application.CalendarEvents.Queries.GetCalendarEvent;
+
+public class GetCalendarEventByIdQueryResult
+{
+    public CalendarEvent CalendarEvent { get; set; } = null!;
+    public List<AttendeeModel> Attendees { get; set; } = null!;
+    public List<EventGuestModel> EventGuests { get; set; } = null!;
```

Comment:
The result is the response body  of the controller and result is using Entity types which can cause cyclical serialisation issue. Entities needs to be converted in to models in the handler or in the controller.